apis/MessagesEndpoint.es6.js

```diff
@@ -0,0 +1,48 @@
+import Comment from '../models2/CommentModel';
+import Post from '../models2/PostModel';
+import Message from '../models2/MessageModel';
+import { runQuery, save } from '../apiBase/APIRequestUtils';
+
+const TYPE_MAP = {
+  't1': Comment,
+  't3': Post,
+  't4': Message,
+};
+
+const parseGetBody = (res, apiResponse, apiRequest) => {
+  const { body } = res;
+  body.data.children.forEach(c => apiResponse.addResult(TYPE_MAP[c.kind].fromJSON(c.data)));
+};
+
+const parsePostBody = (res, apiResponse, apiRequest) => {
+  const { body } = res;
+  console.log(body);
+};
+
+const path = (method, query={}) => {
+  const { subreddit, type, thingId } = query;
+  if (method === 'get') {
+    const sub = subreddit ? `r/${ subreddit }/` : '';
+    return `${ sub }message/${ type }.json`;
+  }
+  if (!thingId) { return 'api/compose'; }
+  return 'api/comment';
```

Comment:
Hitting the comment endpoint seems a little ... off here. Will this case just be actual `Message` models or is this intended for `Comments` / `Posts` that are in your inbox?